ui-participant/src/hub/documents/DocumentLibrary.tsx

```diff
@@ -170,3 +166,74 @@ const fileTypeToIcon = (fileType: string) => {
       return <FontAwesomeIcon className="me-2" icon={faFile}/>
   }
 }
+
+const FileOptionsDropdown = ({ studyEnvParams, participantFile, enrollee, loadDocuments }: {
+  studyEnvParams: StudyEnvParams, participantFile: ParticipantFile, enrollee: Enrollee, loadDocuments: () => void
+}) => {
+  const [showConfirmDelete, setShowConfirmDelete] = useState(false)
+  const { i18n } = useI18n()
+  return (<>
+    <li className="nav-item dropdown d-flex flex-column">
+      <button className="btn btn-outline-primary dropdown-toggle" id="fileOptionsDropdown"
+        data-bs-toggle="dropdown" aria-expanded="false">
+            Options
+      </button>
+      <ul className="dropdown-menu" aria-labelledby="fileOptionsDropdown">
+        <li>
+          <a role={'button'} className="dropdown-item"
+            onClick={() => setShowConfirmDelete(true)}
+          >
+                Delete
+          </a>
+        </li>
+        <li>
+          <a className="dropdown-item" role={'button'} onClick={async () => {
+            const response = await Api.downloadParticipantFile({
+              studyEnvParams, enrolleeShortcode: enrollee.shortcode, fileName: participantFile.fileName
+            })
+            saveBlobAsDownload(await response.blob(), participantFile.fileName)
+          }}>
+            {i18n('documentDownloadButton')}
+          </a>
+        </li>
+      </ul>
+    </li>
+    {showConfirmDelete && <ThemedModal show={true}
+      onHide={() => setShowConfirmDelete(false)} size={'lg'} animation={true}>
+      <Modal.Header>
+        <Modal.Title>
+          <h2 className="fw-bold pb-0 mb-0">
+            {participantFile.associatedAnswers.length === 0 ? 'Are you sure?' : 'This document is in use'}
```

Comment:
should we also check this on the backend? it looks like it naively will just delete it

Reply:
you're right, that would be a good idea. i'll add that check to the backend